utils/freifunk-berlin-wizard-backend/files/rpcd/ffwizard

```diff
@@ -43,10 +47,18 @@ case "$1" in
         fi
         echo '{ "status": "success", "description": "config has been applied" }'
 
-        echo $config > /etc/ffwizard.config.json
+        echo $config > $CONFIG_FILE_PATH
 
         reboot
       ;;
+      getConfig)
+        if [ ! -f $CONFIG_FILE_PATH ]; then
+	  echo '{}'
+	  exit 0
+        fi
+	cat $CONFIG_FILE_PATH
+	exit 0
+      ;;
```

Comment:
This block has mixed indentation. Please change to spaces as in the rest of the file. :)